### PR TITLE
#160577616 Enable users to follow and un-follow others

### PR DESCRIPTION
**Pull request by mutaimwiti**
### What does this PR do?
This PR allows users to follow and un-follow other users. It also allows them to view their own and other users followers and following lists.

### Description of the task to be completed?
**As a User**
I want to be able to follow and or unfollow other authors. 

**Acceptance Criteria**
Scenario 1:
Only **authenticated** users should be able to follow and or unfollow other users.

Scenario 2:
I need a view to manage those I follow and be able to see those who follow me.

### How this should be manually tested?
This PR can be manually tested using Postman.
1. Follow: `POST /api/profiles/<str:username>/follow/`
screen shot 2018-10-14 at 09 46 56

2. Un-follow: `DELETE /api/profiles/<str:username>/follow/`
screen shot 2018-10-14 at 09 46 32

3. Followers: `GET /api/profiles/<str:username>/followers/`
screen shot 2018-10-14 at 09 46 17

4. Following: `GET /api/profiles/<str:username>/following/`
screen shot 2018-10-14 at 09 45 48

### What are the relevant pivotal tracker stories?
[#160577616](https://www.pivotaltracker.com/story/show/160577616)

### CODE REVIEW

**Comment by gitaumoses4** [comment]
Good work on this task.
Looks good to me. I approve. 👍

**Review by gitaumoses4 — APPROVED** [review]
Looks good.

File: authors/apps/profiles/models.py [hunk 1]
```diff
@@ -1,18 +1,57 @@
 from django.db import models
+
+from authors.apps.core.models import TimestampsMixin
 from authors.settings import AUTH_USER_MODEL
 from cloudinary.models import CloudinaryField
 
 
-class Profile(models.Model):
+class FollowMixin(models.Model):
```
Comment (gitaumoses4) [inline authors/apps/profiles/models.py:8]: Good use of multiple inheritance.

**Comment by esirK** [comment]
Fix the Conflicts and we will be good to merge.

**Comment by mutaimwiti** [comment]
> Fix the Conflicts and we will be good to merge.

The conflicts are fixed.

**Pull request merged**